readthedocs/rtd_tests/tests/test_doc_building.py

```diff
@@ -65,7 +65,7 @@ def test_record_command_as_success(self):
             {
                 "build": mock.ANY,
                 "command": command.get_command(),
-                "output": command.output,
+                "output": "",
```

Comment:
command.output was none, but the API needs an empty string. Looks like it was None because the cwd the command was executed in didn't exist, I have fixed it by using /tmp as cwd.